### PR TITLE
[ACS-7582] Fix dialog upload

### PR DESCRIPTION
**Pull request by tomson7777**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

### CODE REVIEW

File: lib/content-services/src/lib/upload/components/file-uploading-dialog.component.html [hunk 1]
```diff
@@ -91,6 +91,7 @@
             mat-button
             *ngIf="canShowCancelAll()"
             (click)="toggleConfirmation()"
+            class="adf-upload-dialog__actions-stop-all-btn"
```
Comment (nikita-web-ua) [inline lib/content-services/src/lib/upload/components/file-uploading-dialog.component.html:94]: do we need this class? I don't where it is used

**Comment by nikita-web-ua** [comment]
For some file extensions image is not visible, and 'close' button font-weight is off 

Screenshot 2024-04-18 at 13 11 17

**Comment by nikita-web-ua** [comment]
> > For some file extensions image is not visible, and 'close' button font-weight is off
> > Screenshot 2024-04-18 at 13 11 17
> 
> what file have you used?

file with `.md` extension

**Comment by MichalKinas** [comment]
Closing as this issue has been fixed by other PR

**Pull request closed without merging**